repos/system_upgrade/common/models/unsafepythonpaths.py

```diff
@@ -0,0 +1,22 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class UnsafePythonPaths(Model):
```

Comment:
It's not descriptive about the content. It does not contain actually any paths and if I am searching for third party python modules, I would not guess that I am interested about this model / message.

Some tips for a possible names (you can come up with a different one):
* ThirdPartyTargetPythonModules
* NonDistributionTargetPythonModules
* ....